### DISCUSSION TITLE
New Woo App

### DISCUSSION THREAD

**Discussion by storeyadev** [woocommerce/woocommerce#51097] [General]
After 13 years of helping thousands of online stores boost their sales, we’re thrilled to align our solutions with the Woo roadmap. 
What are the steps to get our new app listed in the Woo app store?

**Comment by darcie** [comment]
Hey @storeyadev! Glad to hear you're thinking of submitting products to the WooCommerce Marketplace. We're always looking for new products to add that will help store owners! We have a page about how to get started here: https://woocommerce.com/partners/

You can also check out what products we're looking for and further steps to take in our [Getting Started on the Woo Marketplace](https://woocommerce.com/document/marketplace-overview/) guide.

Email `partners @ woocommerce.com` if you have questions before submitting your products!
